_posts/en/newsletters/2024-08-09-newsletter.md

```diff
@@ -0,0 +1,391 @@
+Server][btcpay server repo], [BDK][bdk repo], [Bitcoin Improvement
+Proposals (BIPs)][bips repo], [Lightning BOLTs][bolts repo],
+[Lightning BLIPs][blips repo], [Bitcoin Inquisition][bitcoin inquisition
+repo], and [BINANAs][binana repo]._
+
+- [Bitcoin Core #30493][] enables [full RBF][topic rbf] as the default setting,
+  while leaving the option for node operators to revert to opt-in RBF. Full RBF
+  allows for the replacement of any unconfirmed transaction, regardless of
+  [BIP125][bip125 github] signaling. It has been an option in Bitcoin Core since
+  July 2022 (see Newsletter [#208][news208 fullrbf]), but was previously
+  disabled by default. For the discussions about making full RBF the default,
+  see Newsletter [#263][news263 fullrbf].
+
+- [Bitcoin Core #30285][] adds two key [cluster linearization][wuille cluster]
+  algorithms to the [cluster mempool][topic cluster mempool] project:
+  `MergeLinearizations` for combining two existing linearizations, and
+  `PostLinearize` for improving linearizations by additional processing. This PR
+  builds on work discussed in last week’s Newsletter [#314][news314 cluster].
+
+- [Bitcoin Core #30352][] introduces a new output type, Pay-To-Anchor (P2A), and
+  makes its spending standard. This output type is keyless and enables compact,
+  txid-stable anchors for [CPFP][topic cpfp] fee bumping. Combined with
+  [TRUC][topic v3 transaction relay] transactions, this advances the
+  implementation of [ephemeral anchors][topic ephemeral anchors] to replace
+  [anchor outputs][topic anchor outputs] based on the [CPFP carve-out][topic
+  cpfp carve out] relay rule, for LN commitment transactions.
+
+- [Bitcoin Core #29775][] adds a `testnet4` configuration option that will set
+  the network to [testnet4][topic testnet] as specified in [BIP94][].  Testnet4
+  includes fixes several problems with the previous testnet3 (see [Newsletter
+  #306][news306 testnet]).  The existing Bitcoin Core `testnet` configuration
+  option that uses testnet3 remains available but is expected to be deprecated
+  and removed in subsequent releases.
+
+- [Core Lightning #7476][] catches up to the latest [BOLT12
+  specification][bolt12 spec] updates by adding the rejection of zero-length
+  [blinded paths][topic rv routing] in [offers][topic offers] and invoice
+  requests. Additionally, it allows `offer_issuer_id` to be missing in offers
+  with a provided blinded path. In such cases, the key used to sign the invoice
+  is used as the final blinded path key, since it's safe to assume that the
+  offer issuer has access to this key.
+
+- [Eclair #2884][] implements [BLIP0004][blips endorsement] for [HTLC
+  endorsement][topic htlc endorsement], becoming the first LN implementation to
+  do so, to partially mitigate [channel jamming attacks][topic channel jamming
+  attacks] on the network. This PR enables the relaying of incoming endorsement
+  values, introducing a reputation system for LN where relaying nodes can choose
```

Comment:
Im not sure this suggestion is true, but I think it is and would be an important distinction

```suggestion
  values, introducing a local reputation system for LN where relaying nodes can choose
```

Reply:
Had to rewrite a bit to get the local in there in a way that made sense to me, but I agree that it was an important distinction.